test/OpenTelemetry.Instrumentation.GrpcClient.Tests/OpenTelemetry.Instrumentation.GrpcClient.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Unit test project for OpenTelemetry GrpcClient instrumentation</Description>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Protobuf Include="Proto\greet.proto" GrpcServices="Client" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="Moq" Version="4.11.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
+
+    <PackageReference Include="Google.Protobuf" Version="3.12.3" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.25.0" />
```

Comment:
what is the minimum version of grpc.net.client that we will support with this PR? Need it for the readme/doc.

Reply:
It appears this that 2.23.2 should be the earliest non-prerelease version that this PR supports. Looks like Grpc.Net.Client support was added here: https://github.com/grpc/grpc-dotnet/pull/348. @JamesNK, can you confirm this?

However, I am seeing an issue with the test when running with 2.23.2. There seems to be a race between the time the call ends [here](https://github.com/alanwest/opentelemetry-dotnet/blob/alanwest/grpc-dotnet-instrumentation/test/OpenTelemetry.Instrumentation.GrpcClient.Tests/GrpcClientTests.cs#L76) and the time the `OnStopActivity` gets called [here](https://github.com/alanwest/opentelemetry-dotnet/blob/alanwest/grpc-dotnet-instrumentation/src/OpenTelemetry.Instrumentation.Dependencies/Implementation/GrpcClientDiagnosticListener.cs#L102). Adding a simple delay in the test after the call seems to resolve it, but that seems silly. I'll see if there's something better that can be done here.

Reply:
There are threading issues that have been fixed in earlier versions of the client. I don't think you need to test with it.

All the necessary pieces have been in Grpc.Net.Client since 2.23.2 (this is the first release of the library) so I don't think it needs to be mentioned.